toolchain/check/eval.cpp

```diff
@@ -1767,6 +1767,47 @@ static auto TryEvalInstInContext(EvalContext& eval_context,
       return MakeConstantResult(eval_context.context(), typed_inst, phase);
     }
 
+    case CARBON_KIND(SemIR::RequireCompleteType require_complete): {
```

Comment:
Should we start thinking about splitting longer cases out to their own functions? (this isn't the only one, I don't expect a change here, mainly sounding out your thoughts on this)